README.md

```diff
@@ -14,6 +14,35 @@ Releases details: [CHANGELOG.md](CHANGELOG.md)
     -   id: python-safety-dependencies-check
 ```
 
+
+## How to Use Arguments
+There are a few different arguements that this hook will accept. The first is the `files` arguement. Simply put which file your dependancies are listed in.
```

Comment:
Those options are defined at https://pre-commit.com/#usage

Could you please add a link to that reference page?

Reply:
Perfect!
Thanks for the improved docs 😉